src/Illuminate/Database/Migrations/MigrationRepositoryInterface.php

```diff
@@ -57,6 +62,20 @@ public function createRepository();
     public function repositoryExists();
 
     /**
+     * Determine if the migration repository has the tag column.
+     *
+     * @return bool
+     */
+    public function repositoryTagColumnExists();
```

Comment:
how would you best suggest this is done in a none BC way?

i tried to add it an way thats descriptive, we could just do it in the repositoryExists function to check for the tag column, and then again within the repo install function to check and add if needed without altering the interface.

That being said the migrator calls concrete classes anyway so i doubt the interface is being used externally anyway (could be wrong).